### ISSUE TITLE
Route differentiation in multi-route display

### ISSUE THREAD

**Issue by roata** [roataway/roataway-web#57]
For the multi-route displaying feature to be complete, in terms of visualization, additional changes are needed

- **colour markers**, vehicles belonging to a route will be rendered with a specific colour
- **itinerary** - to indicate the endpoints of the routes
- **vehicle tag** - a label next to the vehicle marker, linking it to a route

Example
![image](https://user-images.githubusercontent.com/48152696/66702478-687c2b00-ed08-11e9-94c0-9204850424c1.png)

Notes on colours and constraints:
- Use colours that are "far" from each other and are easily distinguishable, this [scheme will do](http://colorbrewer2.org/#type=qualitative&scheme=Set1&n=4). ![image](https://user-images.githubusercontent.com/48152696/66702311-88125400-ed06-11e9-852b-59e1c55ac183.png)
- Do not display more than 4 routes at once. First of all, 4 should be enough for anyone, and second - there's no set of 5+ colours that are different enough to be meaningfully differentiated with confidence
- Do not rely on colour as the sole channel to convey the message. Some people are colour blind, and although the chosen schema should address that, writing the actual route name will be an unambiguous answer to the question "what route am I looking at?" (that's what the vehicle tags are for).
- Each colour comes with an accompanying, slightly lighter version, used for the route segments. Here you can see a "blue" for the marker and a "lighter blue" for the route:
![image](https://user-images.githubusercontent.com/48152696/66702437-f0156a00-ed07-11e9-8f0c-243d00441c5e.png)
